src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -3107,105 +3101,104 @@ void TLayout::layout1(Harmony* item, const LayoutContext& ctx)
         ldata->blocks.push_back(TextBlock());
     }
 
-    auto positionPoint = calculateBoundingRect(item, ctx);
+    auto calculateBoundingRect = [](const Harmony* item, Harmony::LayoutData* ldata, const LayoutContext& ctx) -> PointF
+    {
+        const double ypos = (item->placeBelow() && item->staff()) ? item->staff()->height() : 0.0;
+        const FretDiagram* fd = (item->explicitParent() && item->explicitParent()->isFretDiagram())
+                                ? toFretDiagram(item->explicitParent())
+                                : nullptr;
 
-    if (item->hasFrame()) {
-        item->layoutFrame();
-    }
+        const double cw = item->symWidth(SymId::noteheadBlack);
 
-    item->setPos(positionPoint);
-}
+        double newPosX = 0.0;
+        double newPosY = 0.0;
 
-PointF TLayout::calculateBoundingRect(Harmony* item, const LayoutContext& ctx)
-{
-    const Harmony::LayoutData* ldata = item->layoutData();
-    const double ypos = (item->placeBelow() && item->staff()) ? item->staff()->height() : 0.0;
-    const FretDiagram* fd = (item->explicitParent() && item->explicitParent()->isFretDiagram())
-                            ? toFretDiagram(item->explicitParent())
-                            : nullptr;
+        if (item->textList().empty()) {
+            layout1TextBase(item, ldata);
 
-    const double cw = item->symWidth(SymId::noteheadBlack);
+            if (fd) {
+                newPosY = ldata->pos().y();
+            } else {
+                newPosY = ypos - ((item->align() == AlignV::BOTTOM) ? item->harmonyHeight() - ldata->bbox().height() : 0.0);
```

Comment:
Seems to be using the deprecated `harmonyHeight` method here

Reply:
removed, access before set